src/matzah.py

```diff
@@ -122,11 +122,13 @@ def __init__(self, DEVICE, fcSize, numFC, styleDim, outputDim, numBlocks,
         self.Synthesis = SynthesisNetwork(DEVICE, outputDim,
                                           fcSize + numClasses, numBlocks,
                                           startDim, channels, kernel)
+        self.sigmoid = nn.Sigmoid()
 
     def forward(self, styleVec, classVec):
         latentStyle = self.Style.forward(styleVec, classVec)
         # now need to set up the other side of the network.
         generatedCurves = self.Synthesis.forward(latentStyle)
+        generatedCurves = 6 * self.sigmoid(generatedCurves) - 3
```

Comment:
Idk if 3 is the number we should targeting here...  https://github.com/ccurro/font-bakers/blob/master/src/utils.py#L43-L46

But this suffices for now; merging.